eng/tools/spec-gen-sdk-runner/src/command-helpers.ts

```diff
@@ -243,7 +243,7 @@ export function generateArtifact(
 ): number {
   const specGenSdkArtifactName = "spec-gen-sdk-artifact";
   const specGenSdkArtifactFileName = specGenSdkArtifactName + ".json";
-  const specGenSdkArtifactPath = "out/spec-gen-sdk-artifact";
+  const specGenSdkArtifactPath = commandInput.stagedArtifactsFolder!
```

Comment:
`out/spec-gen-sdk-artifact` is a specific artifact called 'spec-gen-sdk-artifact'. Don't change this and you could create another variable to keep 'stagedArtifactsFolder'.

Reply:
Right, I am changing this